packages/components/dropdown/src/use-dropdown.ts

```diff
@@ -119,15 +119,15 @@ export function useDropdown(props: UseDropdownProps) {
   });
 
   const getMenuTriggerProps: PropGetter = (
-    props = {},
+    originalProps = {},
     _ref: Ref<any> | null | undefined = null,
   ) => {
     // These props are not needed for the menu trigger since it is handled by the popover trigger.
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const {onKeyDown, onPress, onPressStart, ...otherMenuTriggerProps} = menuTriggerProps;
 
     return {
```

Comment:
It is important here to use `props.isDisabled` so that the PopoverTrigger component can respect another `isDisabled` gets passed by the user. (Otherwise, the unit test is going to fail.)